### PR TITLE
changes prexis section since we've added autoprfixer

### PR DESCRIPTION
**Pull request by andresgalante**
We've added autoprefixer to our build here:
https://github.com/patternfly/patternfly/pull/430

This PR is changing the prefixes section of the guidelines to reflect that change.

@priley86 @bleathem can you please review?

### CODE REVIEW

File: _includes/css/prefixed-properties.css [hunk 1]
```diff
@@ -1,5 +1,8 @@
-/* Prefixed properties */
+/* Bad example */
 .selector {
   -webkit-box-shadow: 0 1px 2px rgba(0,0,0,.15);
           box-shadow: 0 1px 2px rgba(0,0,0,.15);
 }
+
+/* Good example */
+.selector { box-shadow: 0 1px 2px rgba(0,0,0,.15); }
```
Comment (priley86) [inline _includes/css/prefixed-properties.css:8]: looks good, noting that we have a single declaration (so single line), and you have omitted zero from rgba value (also now in our rules). 👍

File: index.html [hunk 2]
```diff
@@ -273,8 +273,8 @@
 <div class="section" id="css-prefixed-properties">
   <div class="col">
     <h3>Prefixed properties</h3>
-    <p>When using vendor prefixed properties, indent each property such that the declaration's value lines up vertically for easy multi-line editing.</p>
-    <p>In Textmate, use <strong>Text &rarr; Edit Each Line in Selection</strong> (&#8963;&#8984;A). In Sublime Text 2, use <strong>Selection &rarr; Add Previous Line</strong> (&#8963;&#8679;&uarr;) and <strong>Selection &rarr;  Add Next Line</strong> (&#8963;&#8679;&darr;).</p>
+    <p>PatternFly uses <a href="https://github.com/postcss/autoprefixer">Autoprefixer</a> to deal with CSS <a href="https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix">vendor prefixes</a>.</p>
+    <p>Do not add vender prefixes to your code, Autoprefixer will do it for you on build.</p>
```
Comment (priley86) [inline index.html:277]: yep :bowtie:

**Comment by priley86** [comment]
LGTM

File: _includes/css/prefixed-properties.css (hunk 1 above)
Comment (andresgalante) [reply _includes/css/prefixed-properties.css:8]: not here, we need the zeros 😄

**Pull request merged**